package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/apple-sign-in",
-  "version": "0.0.12",
+  "version": "0.0.13",
```

Comment:
No need to bump the version--the plugin maintainer(s) will do this as necessary.